packages/code-du-travail-data/dataset/fiches_ministere_travail/index.js

```diff
@@ -1,7 +1,8 @@
 const fetch = require("node-fetch");
 const JSDOM = require("jsdom").JSDOM;
-
-const FICHES_URLS = require("./ministere-travail-liste-fiches.js");
+const ora = require("ora");
+const { batchPromise } = require("@cdt/dataset...kali/utils");
```

Comment:
what the hell is that ? :p

Reply:
un batcher de requete, ca balance 10requete en // et ca passe au 10 suivantes

Reply:
après, il faudra lui trouvé une meilleur place